source/operations/install-deploy-manage/decommission-server-pool.rst

```diff
@@ -354,3 +352,187 @@ to restart the MinIO service:
 
 Once the deployment is online, use :mc:`mc admin info` to confirm the
 uptime of all remaining servers in the deployment.
+
+.. _minio-decommission-multiple-pools:
+
+Decommission Multiple Server Pools
+----------------------------------
+
+.. versionchanged:: RELEASE.2023-01-18T04-36-38Z
+
+You can decommission multiple server pools from the same command.
+
+MinIO immediately stops write access to all decommissioning pools.
+Decommissioning happens one pool at a time.
+Each pool completes the decommission process before MinIO begins draining the next pool.
```

Comment:
*Oh*. Hm. Nuance.

So you can queue multiple pools for decommissioning in a single command. That might be the best way to describe it. I think there has been internal talk on making it simultaneous (or supporting that), so might as well build the framework.

Also assuming queued but not-started pools still accept writes as normal. Worth checking w/ engineering. If not, then decom multiple pools could significantly hotspot the remaining pools w new write ops while the queued pools go read only.

Reply:
Trying the queueing phrasing.